package.json

```diff
@@ -10,6 +10,7 @@
     "fennec-nightly": "web-ext run --target=firefox-android --source-dir=dist/firefox-mobile --firefox-apk=org.mozilla.fennec_aurora",
     "package:firefox": "web-ext build --source-dir=dist/firefox --artifacts-dir=bin",
     "package:chrome": "npm run build:chrome && crx pack dist/chrome -o bin/webcompat-reporter-chrome.crx",
+    "package:opera": "npm run build:opera && crx pack dist/opera -o bin/webcompat-reporter-opera.crx",
```

Comment:
Can we change the output to `.nex` rather than `.crx`? 

(it makes like zero difference in practice, but that's the file extension opera recognizes natively)